packages/bundle-source/package.json

```diff
@@ -4,6 +4,9 @@
   "description": "Create source bundles from ES Modules",
   "type": "module",
   "main": "src/index.js",
+  "bin": {
+    "bundle-source": "./src/tool.js"
```

Comment:
Since this’ll be called `bundle-source`, let’s take care to commit to the CLI as a non-breaking API. Perhaps caching should be opt-in. We use `commander` elsewhere. We don’t have to use a proper parser now as long as the usage we encourage is forward-compatible.

Reply:
I narrowed the usage:

Force the first argument to be --to so that
 - caching is opt-in
   - support for not caching can come later
 - we can choose other arg patterns later